Week1/homework/8-Type checker.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const y = 60;      //1
+const x = 43;
+const color = 'blue';
+const flag = 'italy';
+
+
+if(typeof y === typeof x){                //2
+    console.log(true);
```

Comment:
The idea of this exercise is to log 'SAME TYPE'  or 'Not the same in all these if statements.